documentation/tutorial.md

```diff
@@ -68,3 +68,145 @@ tutorial overlord    localhost/localhost  2.9.37   unsupported  23:20:53Z
+
+Machine  State    Address       Inst id        Series  AZ  Message
+0        started  10.23.62.156  juju-d35d30-0  focal       Running
+
+```
+Use the IP address listed underneath `Public address` for `mongodb/0` as your host.
+
+#### Retrieving the database name:
+In this case we are logging in via the `admin` user so we will be connecting to the `admin` database. Use `admin` as the database name.
+
+#### Retrieving the replica set name:
+The replica set name is the name of the application. In this tutorial we didn’t use a custom name for the application, so the application name is `mongodb`. You can read more about deploying a charm with a custom name [here](https://juju.is/docs/olm/deploy-a-charm-from-charmhub#heading--override-the-name-of-a-deployed-application). Use `mongodb` as the replica set name.
+
+### Connect via MongoDB URI:
+Now that we have the necessary fields to connect to the URI, we can connect to MongoDB via the URI. Enter the following into the command line, replace the values for  `username`, `password`, `hosts`, `database name`, and the `replica set name` with what you’ve retrieved above:
+```
+mongosh mongodb://<username>:<password>@<hosts>/<database name>?replicaSet=<replica set name>
+```
+
+You should now see:
+```
+Current Mongosh Log ID:	6389e2adec352d5447551ae0
+Connecting to:		mongodb://<credentials>@10.23.62.156/admin?replicaSet=mongodb&appName=mongosh+1.6.1
+Using MongoDB:		5.0.14
+Using Mongosh:		1.6.1
+
+For mongosh info see: https://docs.mongodb.com/mongodb-shell/
+
+
+To help improve our products, anonymous usage data is collected and sent to MongoDB periodically (https://www.mongodb.com/legal/privacy-policy).
+You can opt-out by running the disableTelemetry() command.
+
+------
+   The server generated these startup warnings when booting
+   2022-12-02T11:24:05.416+00:00: Using the XFS filesystem is strongly recommended with the WiredTiger storage engine. See http://dochub.mongodb.org/core/prodnotes-filesystem
+------
+
+------
+   Enable MongoDB's free cloud-based monitoring service, which will then receive and display
+   metrics about your deployment (disk utilization, CPU, operation statistics, etc).
+
+   The monitoring data will be available on a MongoDB website with a unique URL accessible to you
+   and anyone you share the URL with. MongoDB may use this information to make product
+   improvements and to suggest MongoDB products and deployment options to you.
+
+   To enable free monitoring, run the following command: db.enableFreeMonitoring()
+   To permanently disable this reminder, run the following command: db.disableFreeMonitoring()
+------
+
+mongodb [primary] admin>
+```
+
+You can now interact with MongoDB directly using any [MongoDB commands](https://www.mongodb.com/docs/manual/reference/command/). For example entering `show dbs` should output something like:
+```
+admin   172.00 KiB
+config  120.00 KiB
+local   404.00 KiB
+```
+
```

Comment:
I would just type 3 commands, like:

1. Creating a new "test" database
2. Add a user
3. Create a collection with RW operations for the user

Reply:
Minor tweak: MongoDB wise and tutorial wise I think it will make more sense to add a user which has RW operations over a database not a collection. So I'll modify this a smidge